tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py

```diff
@@ -239,6 +240,46 @@ def assertSuccessfulRpcs(self,
             0,
             msg=f'Expected all RPCs to succeed: {failed} of {num_rpcs} failed')
 
+    @staticmethod
+    def diffAccumulatedStatsPerMethod(
+            before: grpc_testing.LoadBalancerAccumulatedStatsResponse,
+            after: grpc_testing.LoadBalancerAccumulatedStatsResponse):
+        """Only diffs stats_per_method, as the other fields are deprecated."""
+        diff = grpc_testing.LoadBalancerAccumulatedStatsResponse()
+        for method, method_stats in after.stats_per_method.items():
+            for status, count in method_stats.result.items():
+                count -= before.stats_per_method[method].result[status]
+                if count < 0:
+                    raise AssertionError("Diff of count shouldn't be negative")
```

Comment:
Alternatively,  you can do
```suggestion
                    self.assertGreaterEqual(count, 0, "Diff of count shouldn't be negative")
```

The benefit you error message will be extended with

```
"3" unexpectedly not greater than or equal to "0"
```

which is nice, because it prints the count.

But you'd have to remove `@staticmethod`.